x-pack/plugins/apm/server/routes/apm_routes/register_apm_server_routes.ts

```diff
@@ -181,6 +182,9 @@ export function registerRoutes({
             opts.statusCode = error.output.statusCode;
           }
 
+          // capture error with APM node agent
+          apm.captureError(error);
```

Comment:
Do we want to skip RequestAbortedError? Also, can we get the stack trace / message fixes on the platform side so it's not duplicated? Is there any other way around it?

Reply:
> Do we want to skip RequestAbortedError?
I think capturing aborted requests will mostly just be noise but open to change the if you think.

> Also, can we get the stack trace / message fixed on the platform side so it's not duplicated? Is there any other way around it?

Fixing this on the platform side is preferably but I don't think it'll happen for 8.7 so I think this is still an improvement over status quo. I can do a follow up for 8.8.